src/tools/kg_tools.py

```
@@ -21,71 +26,197 @@ async def you_trippin(ctx: Context, script_path: str) -> str:
+            message="Script validation completed",
+            operation_id=operation_id
+        )
     except Exception as e:
-        return json.dumps({"success": False, "error": str(e)})
+        error_handler = MCPErrorHandler("you_trippin", operation_id)
+        if "FileNotFoundError" in str(type(e)):
+            error = error_handler.create_error(
+                e,
+                category=ErrorCategory.FILESYSTEM,
+                error_code="SCRIPT_NOT_FOUND",
+                context={"script_path": script_path}
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)